neqo-transport/src/srt.rs

```diff
@@ -0,0 +1,118 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Stateless Reset Token implementation.
+
+use neqo_crypto::random;
+
+use crate::Error;
+
+pub const SRT_LEN: usize = 16;
```

Comment:
```suggestion
pub const LEN: usize = 16;
```

Already in the `srt` module. No need to prefix with the module name.